app/views/account/users/_user.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td= link_to user.email, user.email
+  td= user.name
+  td
+    - if current_user.admin?
+      = simple_form_for(user, url: account_user_path(user)) do |f|
```

Comment:
The main point is that we don't have what to edit except user's role and banning user. So maybe "edit user" form isn't necessary?

It is easier to do these operations from users list